src/offerings.ts

```diff
@@ -325,3 +380,181 @@ export enum PRORATION_MODE {
      */
     IMMEDIATE_AND_CHARGE_FULL_PRICE = 5,
 }
+
+/**
+ * Contains all details associated with a SubscriptionOption
+ * Used only for Google
+ */
+export interface SubscriptionOption {
+    /**
+     * Identifier of the subscription option
+     * If this SubscriptionOption represents a base plan, this will be the basePlanId.
+     * If it represents an offer, it will be {basePlanId}:{offerId}
+     */
+    readonly id: string;
+
+    /**
+     * Identifier of the StoreProduct associated with this SubscriptionOption
+     * This will be {subId}:{basePlanId}
+     */
+    readonly storeProductId: string;
+
+    /**
+     * Identifer of the subscription associated with this SubscriptionOption
+     * This will be {subId}
+     */
+    readonly productId: string;
+
+    /**
+     * Pricing phases defining a user's payment plan for the product over time.
+     */
+    readonly pricingPhases: PricingPhase[];
+
+    /**
+     * Tags defined on the base plan or offer. Empty for Amazon.
+     */
+    readonly tags: string[];
+
+    /**
+     * True if this SubscriptionOption represents a subscription base plan (rather than an offer).
+     */
+    readonly isBasePlan: boolean;
+
+    /**
+     * The subscription period of fullPricePhase (after free and intro trials).
+     */
+    readonly billingPeriod: Period | null;
```

Comment:
Not a blocker but I'm sometimes confused since in some places we add the `?` to the name of the member and others we do the `| null`.

Reply:
`?` means `undefined`, which is not the same as `null`. In this case, there will always be a `billingPeriod` but it sometimes can be `null`.

Reply:
Ahh yeah 🤦 I always forget. Thanks!